File: Relaciones/ejercicio 2/Ropa.java
package relaciones;
import java.util.*;
public class Ropa { 
    private String tipo; 
    private String material; 
 
    public Ropa(String tipo, String material) { 
        this.tipo = tipo; 
        this.material = material; 
    } 
 
    public void mostrar() { 
        System.out.println("Ropa -> Tipo: " + tipo + ", Material: " + material); 
    } 

    public String getTipo() {
        return tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    public String getMaterial() {
        return material;
    }

    public void setMaterial(String material) {
        this.material = material;
    }
}

File: Relaciones/ejercicio 2/Ropa.py
class Ropa:
    def __init__(self, tipo, material):
        self.tipo = tipo
        self.material = material

    def mostrar(self):
        print(f"Ropa -> Tipo: {self.tipo}, Material: {self.material}")

    def getTipo(self):
        return self.tipo

    def setTipo(self, tipo):
        self.tipo = tipo

    def getMaterial(self):
        return self.material

    def setMaterial(self, material):
        self.material = material